src/app/src/pages/Comments.tsx

```diff
@@ -0,0 +1,277 @@
+          />
+          <EditContainer>
+            <Block>
+              <Header>Add Comment</Header>
+              <FormControl variant="standard" sx={formStyle}>
+                <InputLabel id="demo-simple-select-standard-label" required>
+                  Comment type
+                </InputLabel>
+                <Select
+                  labelId="demo-simple-select-standard-label"
+                  id="demo-simple-select-standard"
+                  value={type}
+                  onChange={handleType}
+                  label="Type"
+                >
+                  <MenuItem value="text/plain">Plain</MenuItem>
+                  <MenuItem value="text/markdown">Markdown</MenuItem>
+                </Select>
+              </FormControl>
+              <Content>
+                {" "}
+                <WriteOrPreview>
+                  <ButtonGroup
+                    variant="text"
+                    color="inherit"
+                    size="large"
+                    sx={{ p: 1, borderBottom: "1px solid black" }}
+                  >
+                    <CustomButton
+                      onClick={() => setOpenWrite(true)}
+                      sx={{ background: openWrite ? "#b5b5b5" : "white" }}
+                    >
+                      {" "}
+                      Write{" "}
+                    </CustomButton>
+                    <CustomButton
+                      onClick={() => setOpenWrite(false)}
+                      sx={{ background: !openWrite ? "#b5b5b5" : "white" }}
+                    >
+                      {" "}
+                      Preview
+                    </CustomButton>
+                  </ButtonGroup>
+                  <ActualContent>
+                    {openWrite ? (
+                      <TextField
+                        required
+                        id="multiline-flexible"
+                        label="Content"
+                        multiline
+                        fullWidth
+                        maxRows={10}
+                        value={content}
+                        onChange={handleTextChange}
+                      />
+                    ) : type === "text/markdown" ? (
+                      <ReactMarkdown>{content}</ReactMarkdown>
+                    ) : (
+                      content
+                    )}
+                  </ActualContent>
+                </WriteOrPreview>
+              </Content>
+            </Block>
+            <Fab
+              color="primary"
+              aria-label="check"
+              sx={{
+                color: "black",
+                background: "#46ECA6",
+                "&:hover": { background: "#18E78F" },
+              }}
+            >
+              <CheckIcon onClick={createComment} />
+            </Fab>
+          </EditContainer>
+        </Backdrop>
+      ) : null}
+      {post ? (
+        <UserPost post={post} likes={5} currentUser={currentUser} postAuthor={currentUser} />
```

Comment:
We want the authorID author as postAuthor (use authorID to get author), we're not always commenting on our own posts.